pysat/tests/test_utils.py

```diff
@@ -321,7 +321,13 @@ def teardown(self):
     def eval_loaded_data(self):
         """Evaluate loaded test data."""
         # Test that the written and loaded data matches the initial data
-        for dkey in self.testInst.data.columns:
+        if self.testInst.pandas_format:
```

Comment:
```suggestion
       for dkey in self.testInst.variables:
```
should work for both pandas and xarray.

Reply:
The `loaded_inst` object varies, so we still need the if/else statement.

Reply:
I tested moving the `keys` eval outside the if/else loop and it failed.